util/settings/settings.go

```diff
@@ -106,6 +106,8 @@ type RepoCredentials struct {
 	Insecure bool `json:"insecure,omitempty"`
 	// Whether the repo is git-lfs enabled
 	EnableLFS bool `json:"enableLfs,omitempty"`
+	// Refspecs to be used for fetching revisions from the remote repository
+	FetchRefspecs []string `json:"fetchRefspecs,omitempty"`
```

Comment:
I think we have changed how we do just, just provide `func (mrg *SettingsManager) FetchRefspecs []string`. See `GetKustomizeBuildOptions` for example.

Reply:
Not sure what you mean. Do you suggest to convert the fetch refspecs to be a global configuration? Right now, they're stored per repo...